paddle/fluid/framework/ir/conv_elementwise_add_mkldnn_fuse_pass.cc

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/framework/ir/conv_elementwise_add_mkldnn_fuse_pass.h"
+#include <functional>
+
+#include "paddle/fluid/framework/ir/graph_traits.h"
+
+namespace paddle {
+namespace framework {
+namespace ir {
+namespace patterns {
+
+struct Pattern : public PatternBase {
+  Pattern(PDPattern* pattern, const std::string& name_scope)
+      : PatternBase{pattern, name_scope, ""} {}
+
+ private:
+  std::string name_scope() { return name_scope_; }
```

Comment:
These APIs are from PatternBase, seems no need to redefine them, just use them is OK.